openjpa-jdbc/src/main/java/org/apache/openjpa/jdbc/schema/DBCPDriverDataSource.java

```diff
@@ -194,8 +194,8 @@ private Properties updateDBCPProperties(Properties props) {
         if (hasKey(dbcpProps, "minIdle") == null) {
             dbcpProps.setProperty("minIdle", "0");
         }
-        if (hasKey(dbcpProps, "maxActive") == null) {
```

Comment:
Probably we should keep both? In which case it should work with both old and new commons-dbcp libs, isn't?
Of course this needs some comment to explain the reason and the change which made it necessary.

Reply:
Hmmmm. Probably true. Those properties get applied via reflection anyway. So every method which exists on a ConnectionFactory will get invoked afair. But we should cross check this.

Probably we might even remove all that stuff? Because most of the pools have sensible defaults anyway, right?

Reply:
+1, it sounds worse to corrupt pool defaults and not enable the user to use the pool doc anymore

Reply:
So I'll drop `SimpleDriverDataSource`, `DBCPDriverDataSource` and `AutoDriverDataSource`, and will check nothing is broken
Would it be sufficient?

Reply:
Should `DriverDataSource` also be dropped?